File: miraculous-frontend/miraculous-ui/src/dto/album-like-dto/album-like-dto.ts
export class AlbumLikeDto {

  siteVisitorId: number;
  albumId: number;
  numberOfLikes: number;
  albumLikedByCurrentVisitor: boolean;


  constructor(siteVisitorId?: number, albumId?: number, numberOfLikes?: number, isAlbumLikedByCurrentVisitor?: boolean) {
    this.siteVisitorId = siteVisitorId;
    this.albumId = albumId;
    this.numberOfLikes = numberOfLikes;
    this.albumLikedByCurrentVisitor = isAlbumLikedByCurrentVisitor;
  }
}

File: miraculous-backend/src/main/java/com/example/miraculousbackend/dto/AlbumLikeDto.java
package com.example.miraculousbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AlbumLikeDto {

    private long siteVisitorId;
    private long albumId;
    private long numberOfLikes;
    private boolean isAlbumLikedByCurrentVisitor;
}
